test/integration_test.sh

```diff
@@ -232,18 +230,14 @@ elif [ $ss2_mode == "local" ]; then
 fi
 
 # 6. Create config.json
-# (TODO: Use Henry's script here)
-# TODO: use config file from config repo
-# dev_secrets.json will come from Vault eventually
 printf "\n\nCreating Lira config"
-printf "\nUsing $env_config_json"
-printf "\nUsing $secrets_json\n"
-python $script_dir/create_lira_config.py \
-    --env_config_file $env_config_json \
-    --secrets_file $secrets_json \
-    --tenx_prefix $tenx_prefix \
-    --ss2_prefix $ss2_prefix \
-    --pipeline_tools_prefix $pipeline_tools_prefix > config.json
+docker run -it --rm -v $PWD:/working -e VAULT_TOKEN=$vault_token \
```

Comment:
Getting an error here. I was running from a subdirectory that I made: secondary-analysis/test/test-runs. I got farther by changing this to ```docker run -it --rm -v $repo_dir:/working -v $work_dir:/out -e VAULT_TOKEN=$vault_token \``` (I also set repo_dir in the script to be the root of the secondary-analysis repo) and then below setting ```OUT_PATH=/out```. But I then ran into some vault auth errors which are probably due to something I'm doing wrong.